gandalf/src/main/java/io/github/btkelly/gandalf/activities/GandalfActivity.java

```diff
@@ -69,32 +67,40 @@ protected final void onResume() {
 
     @Override
     public final void onRequiredUpdate(RequiredUpdate requiredUpdate) {
-        //TODO show in a dialog using dialog util
-        //This is just temporary until this is implemented
-        new AlertDialog.Builder(this)
-                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
-                    @Override
-                    public void onClick(DialogInterface dialog, int which) {
-                        Log.e("IGNORE", gandalf.toString());
-                    }
-                })
-                .show();
+        BootstrapDialogUtil.showRequiredUpdateDialog(
+                this,
+                this.gandalf,
+                requiredUpdate
+        );
     }
 
     @Override
     public final void onOptionalUpdate(OptionalUpdate optionalUpdate) {
-        //TODO show in a dialog using dialog util
-        this.gandalf.save(optionalUpdate);
+        BootstrapDialogUtil.showOptionalUpdateDialog(
+                this,
+                this.gandalf,
+                optionalUpdate,
+                this
+        );
     }
 
     @Override
     public final void onAlert(Alert alert) {
-        //TODO show in a dialog using dialog util
-        this.gandalf.save(alert);
+        BootstrapDialogUtil.showAlertDialog(
+                this,
+                this.gandalf,
+                alert,
+                this
+        );
     }
 
     @Override
     public final void onNoActionRequired() {
         youShallPass();
     }
+
+    @Override
+    public void continueLoading() {
+        youShallPass();
```

Comment:
Discussed in Slack; decided we can assess the value of separate cases more easily when the library has matured a bit.